cfme/tests/automate/custom_button/test_infra_objects.py

```diff
@@ -168,13 +174,19 @@ def test_custom_button_automate(appliance, request, submit, setup_obj, button_gr
             except AttributeError:
                 paginator = view.entities.paginator
 
-            entity_count = paginator.items_amount
+            if paginator.items_amount > paginator.items_per_page:
+                entity_count = paginator.items_per_page
+            else:
+                entity_count = paginator.items_amount
             paginator.check_all()
         else:
             entity_count = 1
 
-        # First collect number of request already available in automation log
-        initial_request_count = check_log_requests_count(appliance)
+        # Clear the automation log
+        clean = appliance.ssh_client.run_command('echo -n "" > '
+                                                 '/var/www/miq/vmdb/log/automation.log')
+        assert clean.success
```

Comment:
Consider just asserting against the `run_command` return directly since its not re-used.

`assert appliance.ssh_client.run_command('stuff')`